src/libcore/simd.rs

```diff
@@ -37,11 +24,33 @@
 #![unstable(feature = "core_simd",
             reason = "needs an RFC to flesh out the design",
             issue = "27731")]
+#![deprecated(since = "1.3.0",
+              reason = "use the external `simd` crate instead")]
 
 #![allow(non_camel_case_types)]
 #![allow(missing_docs)]
+#![allow(deprecated)]
+
+use ops::{Add, Sub, Mul, Div, Shl, Shr, BitAnd, BitOr, BitXor};
+
+macro_rules! argh {
```

Comment:
Oh wow, this is a very neat trick, it should be part of our usual bootstrapping toolbox!